scripts/publish.js

```diff
@@ -100,7 +100,7 @@ if (!/latest/.test(DOCKER_IMAGE_VERSION)) {
   shell.exec(`echo ${DOCKER_IMAGE_VERSION} > version.txt`);
   shell.exec(`git add version.txt`);
   shell.exec(`git commit -a -m "Publish version: ${DOCKER_IMAGE_VERSION}"`);
-  shell.exec(`git push`);
+  shell.exec(`git push --set-upstream origin ${currentBranch}`);
```

Comment:
Because we are in a new CI env each time, we may need to set the upstream branch as if it was the first time pushing.